### ISSUE TITLE
An error is raised in Tapioca using ffaker

### ISSUE THREAD

**Issue by N1koKouloufakos** [ffaker/ffaker#531]
## What
When using `bin/tapioca gem ffaker`, the rbi files are not generated and an error is raised because the VERSION constant is not loaded correctly.
```
/Users/my_user/.gem/ruby/3.2.1/gems/ffaker-2.21.0/lib/ffaker/utils/module_utils.rb:21:in `read': No such file or directory @ rb_sysopen - /Users/my_user/.gem/ruby/3.2.1/gems/ffaker-2.21.0/lib/ffaker/data/f_faker/version (Errno::ENOENT)
      from /Users/my_user/.gem/ruby/3.2.1/gems/ffaker-2.21.0/lib/ffaker/utils/module_utils.rb:21:in `const_missing'
      from /Users/my_user/.gem/ruby/3.2.1/gems/tapioca-0.11.5/lib/tapioca/runtime/reflection.rb:50:in `const_get'
      from /Users/my_user/.gem/ruby/3.2.1/gems/tapioca-0.11.5/lib/tapioca/runtime/reflection.rb:50:in `constantize'
      from /Users/my_user/.gem/ruby/3.2.1/gems/tapioca-0.11.5/lib/tapioca/runtime/trackers/autoload.rb:25:in `block in eager_load_all!'
      from /Users/my_user/.gem/ruby/3.2.1/gems/tapioca-0.11.5/lib/tapioca/runtime/trackers/autoload.rb:50:in `with_disabled_exits'
 ```

This [appears to be fixed](https://github.com/ffaker/ffaker/commit/87671bd1627e54a67a9a81d07223bb6b038555c6#diff-421d693fc670ec61fbb154640064166e48203fd3fad0d8c2cede1df18a842d68R218) in `main`, so when we pin our version to main it works fine.

## Solution
I think we just need a version bump, please.

cc: @brandoncc

**Comment by marocchino** [comment]
I just released v2.22.0

**Comment by AlexWayfer** [comment]
> I just released v2.22.0

I believe it's a big mistake because you have published a new **minor** version with **breaking** changes, like required Ruby versions dropping.

**Comment by marocchino** [comment]
but.. I consider other drop version as minor before this.😅
